server/country/countryService.js

```diff
@@ -19,24 +20,43 @@ const getCountryConfig = async (countryIso, schemaName = 'public') => {
 const getCountryConfigFull = async (countryIso, schemaName = 'public') => {
   const [config, result] = await Promise.all([
     getCountryConfig(countryIso, schemaName),
-    traditionalTableRepository.read(countryIso, 'climaticDomain', schemaName)
+    traditionalTableRepository.read(countryIso, 'climaticDomain', schemaName),
   ])
 
   const climaticDomainPercents = {
-    boreal: (result && result[0][0]) || config.climaticDomainPercents2015 && config.climaticDomainPercents2015.boreal,
-    temperate: (result && result[1][0]) || config.climaticDomainPercents2015 && config.climaticDomainPercents2015.temperate,
-    subtropical: (result && result[2][0]) || config.climaticDomainPercents2015 && config.climaticDomainPercents2015.subtropical,
-    tropical: (result && result[3][0]) || config.climaticDomainPercents2015 && config.climaticDomainPercents2015.tropical,
+    boreal: (result && result[0][0]) || (config.climaticDomainPercents2015 && config.climaticDomainPercents2015.boreal),
+    temperate:
+      (result && result[1][0]) || (config.climaticDomainPercents2015 && config.climaticDomainPercents2015.temperate),
+    subtropical:
+      (result && result[2][0]) || (config.climaticDomainPercents2015 && config.climaticDomainPercents2015.subtropical),
+    tropical:
+      (result && result[3][0]) || (config.climaticDomainPercents2015 && config.climaticDomainPercents2015.tropical),
   }
 
   config.climaticDomainPercents = climaticDomainPercents
 
   return config
+}
 
+const getAllCountriesList = async () => {
+  const allCountries = await countryRepository.getAllCountriesList()
+  const allDeskStudies = await assessmentRepository.getAllDeskStudies()
+  const deskStudies = {}
+  allDeskStudies.forEach(({ countryIso, deskStudy }) => {
+    deskStudies[countryIso] = deskStudy
+  })
+
+  return allCountries.map((country) => ({
+    ...country,
+    assessment: {
+      fra: {
+        deskStudy: deskStudies[country.countryIso],
+      },
```

Comment:
better if this object is associated directly when fetching allCountries, rather than fetching the desk study ones